corehq/apps/data_analytics/models.py

```diff
@@ -1,6 +1,15 @@
+from collections import namedtuple
+
 from django.db import models
 
-from corehq.apps.app_manager.const import AMPLIFIES_YES, AMPLIFIES_NO, AMPLIFIES_NOT_SET
+from corehq.apps.data_analytics.const import NOT_SET
+
+
+GIRExportRow = namedtuple('girrow',
```

Comment:
nit: I think it's clearer if the name is `GIRExportRow` so that way it will be consistent in code and, e.g. `repr()`